protocols/gossipsub/src/behaviour/tests.rs

```diff
@@ -660,7 +669,11 @@ fn test_publish_without_flood_publishing() {
         .iter()
         .fold(vec![], |mut collected_publish, e| match e {
             ToSwarm::NotifyHandler {
-                event: HandlerIn::Message(ref message),
+                event:
+                    HandlerIn::Message {
+                        rpc_type: RpcType::Forward,
```

Comment:
this particular one (`test_publish_without_flood_publishing`) should instead be `RpcType::Publish` right? asking cause the method called by the `NetworkBehaviour` is `forward_msg`